### ISSUE TITLE
ol.style.Icon doubles when using vector Context.drawFeature

### ISSUE THREAD

**Issue by DmitryKamynin** [openlayers/openlayers#13954]
I was trying to make a ripple animation around a point with style.icon applied on the point. In order for my pulsation to be under the icon, I draw the pulsations as it should in postrender first and then overlay the icon on top. But my icon, instead of being the original view, is slightly reduced and shifted relative to the center of the point, from which we see 2 icons

1. Go to http://jsfiddle.net/upm2sxbq/20/
2. You will see how the icon shrinks and shifts relative to the center

I expected that vectorContext.drawFeature would just overlay its icon on top and everything would look acceptable.

**Comment by mike-000** [comment]
When I copy you code and run it outside of jsfiddle it works correctly

**Comment by mike-000** [comment]
I can now reproduce the problem, vector context icons are misplaced and too small if the pixel ratio is greater than 1.  https://codesandbox.io/s/icon-scale-forked-rdxh41  It works correctly up to version 6.14 but not in 6.15.

**Comment by DmitryKamynin** [comment]
I didn't understand your answer much, I'm not good at canvas, could you rephrase your answer
